graph_theory/helpers.py

```diff
@@ -5,7 +5,7 @@
 
 def create_random_map(min_length: int = 5, max_length: int = 10) -> list[list[int]]:
     """
-    Creates a random map of 0's and 1's (water/land) with a random number of rows and columns
+    Create a random map of 0's and 1's (water/land) with a random number of rows and columns
```

Comment:
What is a purpose for changing it?

Reply:
I found a guide for docstrings saying that verbs should be in the infinitive form, but I think it's up to us if we want to keep it in third-person or change to infinitive.
Link: https://pandas.pydata.org/docs/development/contributing_docstring.html#section-1-short-summary

![image](https://user-images.githubusercontent.com/62672370/200551008-7abfa800-4981-49d0-9f27-3a3c2ec98d54.png)